web/src/components/organisms/Project/settings/index.tsx

```diff
@@ -44,7 +44,6 @@ const ProjectSettings: React.FC = () => {
           name: values.name,
           description: values.description,
         });
-        form.resetFields();
```

Comment:
When you remove this, doesn't this mean after submitting the form will NEVER reset?

Reply:
@KaWaite , yes and this is intentional because it is an update form, it should keep the same of info.